UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.API/Controllers/ProductDataController.cs

```diff
@@ -10,15 +10,14 @@
 using System.Threading.Tasks;
 using UKHO.ExchangeSetService.API.Extensions;
 using UKHO.ExchangeSetService.API.Services;
-using UKHO.ExchangeSetService.Common.Helpers;
 using UKHO.ExchangeSetService.Common.Logging;
 using UKHO.ExchangeSetService.Common.Models.Request;
 using UKHO.ExchangeSetService.Common.Models.Response;
 
 namespace UKHO.ExchangeSetService.API.Controllers
 {
     [ApiController]
-    [Authorize(Roles = ApplicationRoles.ExchangeSetServiceUser)]
```

Comment:
Request you to remove ApplicationRoles.cs as we are no longer using it.

Reply:
Resolved. Removed ApplicationRoles.cs.